pkgs/by-name/do/doppler/package.nix

```
@@ -26,8 +27,9 @@ buildGoModule rec {
 
   nativeBuildInputs = [ installShellFiles ];
 
-  postInstall = ''
+  postInstall = lib.optionalString (stdenv.hostPlatform.system != stdenv.targetPlatform.system) ''
```

Thanks for the eval fix. This is more correct:

```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```